Quote_Manager_CLI_Peter/var/log/quote_manager_error.log

```diff
@@ -0,0 +1,163 @@
+Error occurred during database operation: Could not locate column in row for column 'author'
```

Comment:
you don't need to commit your log file, it's a going to be larger in the future and for security reasons it isn't safe because you might log sensitive data and that'd be exposed via commits